lib/flagship.dart

```diff
@@ -104,4 +109,12 @@ class Flagship {
   static void setLoggerLevel(Level newLevel) {
     LogManager.level = newLevel;
   }
+
+  static Status getStatus() {
+    return Flagship._singleton._status;
+  }
+
+  static updateState(Status state) {
```

Comment:
This method is callable by the developer, no ? It should not be (the user of the SDK should not be able to set the state)

Reply:
Ohh Non should not be visible for the user .

Reply:
Fixed